style.css

```diff
@@ -130,90 +130,509 @@ header {
-    font-size: 20px;
-    line-height: 28px;
-    color: #344563;
-    flex: none;
-    order: 1;
-    flex-grow: 0;
-  }
+/* Cards works container */
+.cards_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  left: -1%;
+  top: 718px;
+}
+
+.card_works {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 542px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card_right h2 {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card_right img {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+  margin-top: 5px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card_right p {
+  width: 295px;
+  height: 76px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 2%;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.language_row ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 214px;
+  height: 24px;
+  margin-top: 10px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.html {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 30px;
+  height: 13px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.css {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 24px;
+  height: 13px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.javascript {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 67px;
+  height: 13px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+/* See Project section */
+.see_button a {
+  width: 102px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.see_button {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  position: absolute;
+  width: 126px;
+  height: 48px;
+  left: 64px;
+  margin-top: 22px;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
```

Comment:
- [ ] One of the design requirements is to implement the button interaction for users. That means you should implement the pseudo-state for `hover`, `active` and `disabled` for the button in the cards. To use the correct CSS declarations for the various button interactions, kindly click on the design elements on this part of the Figma template provided:

![image](https://user-images.githubusercontent.com/77106962/234288165-eb3d7039-0961-4bf8-97a5-e1c54d6e1a09.png)

Kindly implement this change 👍 😎. You can learn more about pseudo-classes for buttons by reading this [article](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes). 💻